engine/src/main/java/org/terasology/logic/chat/ChatSystem.java

```diff
@@ -172,4 +172,4 @@ public String whisper(
 
         return senderMessage;
     }
-}
+}
```

Comment:
Best practice: Leave a single empty new line after the final `}` (otherwise GitHub will give you that angry red symbol)